bayesian/utility/user_utils.py

```diff
@@ -0,0 +1,66 @@
+"""Definition of all utility and db interactions for user management."""
+import datetime
+import logging
+
+from sqlalchemy.exc import SQLAlchemyError
+from sqlalchemy import insert, update
+
+from f8a_worker.models import (UserDetails)
+from bayesian import rdb
+
+logger = logging.getLogger(__file__)
+
+
+def get_user(uuid_val):
+    """Get User."""
+    try:
+        query = rdb.session.query(UserDetails).filter(UserDetails.user_id == uuid_val)
+        user = query.one()
+        return user
+    except SQLAlchemyError:
+        rdb.session.rollback()
+        logger.exception(f"Error fetching user with id {uuid_val}")
+        raise UserException("Error fetching user")
```

Comment:
Follow `raise Exception(...) from e` pattern to preserve actual call trace when rethrowing an exception

Reply:
Done